### PR TITLE
[DNM] update readme to test mergify

### PR DESCRIPTION
**Pull request by nixpanic**
# Describe what this PR does #

Testing updated mergify rules.

When someone approves this PR, and a subsequent update is made, the approval should be removed. Any requests for changes should be kept, and not dropped.

## Is there anything that requires special attention ##

After testing, this PR should be closed without getting merged.

## Related issues ##

See-also: #583

### CODE REVIEW

**Comment by Madhu-1** [comment]
@nixpanic approved can you update the PR to verify it

**Comment by nixpanic** [comment]
@Madhu-1 can you leave a review to request changes too? After updating that negative review should stick around.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Ceph CSI
 
+**TESTING TO DROP APPROVALS**
+
+***And now the approval from Madhu should get dropped***
```
Comment (Madhu-1) [inline README.md]: testing request  changes

**Comment by Madhu-1** [comment]
@nixpanic done

**Comment by Madhu-1** [comment]
@nixpanic I think its working as expected, Thanks for the  Fix :+1

**Comment by nixpanic** [comment]
It looks to be working as expected. Dismissing a approval happened within a minute after updating the PR. The 'changes requested' review is still there after 10 minutes.

**Pull request closed without merging**